src/main/java/com/mycompany/hangman/actions/ActionRepository.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.mycompany.hangman.actions;
+
+import com.mycompany.hangman.repo.Repository;
+import javax.swing.Action;
+
+/**
+ *
+ * @author Cory
+ */
+public interface ActionRepository extends Repository<Action>

```

Comment:
to me a “repository” makes more sense when storing “domain data”, not so much UI elements. an example of domain data in this case could be if a user wants to save a game session to continue later. or if they want to store settings (e.g. how many attempts to allow, complexity of words, language to use, etc). i’d suggest using the repo pattern for that type of purpose instead

Reply:
iow, “domain data” are things the end-users have control over